spec/models/restitution/inventaire/organisation_methode_spec.rb

```diff
@@ -4,44 +4,93 @@
+        ).to eql(Competence::NIVEAU_4)
+      end
+    end
 
-  context 'en réussite et moins de 70 ouvertures de contenants' do
-    it 'a le niveau 4' do
-      expect(restitution).to receive(:reussite?).and_return(true)
-      expect(restitution).to receive(:nombre_ouverture_contenant).and_return(60)
-      expect(
-        described_class.new(restitution).niveau
-      ).to eql(Competence::NIVEAU_4)
+    context 'en réussite et entre de 70 et 120 ouvertures de contenants' do
+      it 'a le niveau 3' do
+        expect(restitution).to receive(:reussite?).and_return(true)
+        expect(restitution).to receive(:version?).and_return(false)
+        expect(restitution).to receive(:nombre_ouverture_contenant).and_return(80)
+        expect(
+          described_class.new(restitution).niveau
+        ).to eql(Competence::NIVEAU_3)
+      end
     end
-  end
 
-  context 'en réussite et entre de 70 et 120 ouvertures de contenants' do
-    it 'a le niveau 3' do
-      expect(restitution).to receive(:reussite?).and_return(true)
-      expect(restitution).to receive(:nombre_ouverture_contenant).and_return(80)
-      expect(
-        described_class.new(restitution).niveau
-      ).to eql(Competence::NIVEAU_3)
+    context 'en réussite et entre de 120 et 250 ouvertures de contenants' do
+      it 'a le niveau 2' do
+        expect(restitution).to receive(:reussite?).and_return(true)
+        expect(restitution).to receive(:version?).and_return(false)
+        expect(restitution).to receive(:nombre_ouverture_contenant).and_return(200)
+        expect(
+          described_class.new(restitution).niveau
+        ).to eql(Competence::NIVEAU_2)
+      end
     end
-  end
 
-  context 'en réussite et entre de 120 et 250 ouvertures de contenants' do
-    it 'a le niveau 2' do
-      expect(restitution).to receive(:reussite?).and_return(true)
-      expect(restitution).to receive(:nombre_ouverture_contenant).and_return(200)
-      expect(
-        described_class.new(restitution).niveau
-      ).to eql(Competence::NIVEAU_2)
+    context 'en réussite et plus de 250 ouvertures de contenants' do
+      it 'a le niveau 2' do
+        expect(restitution).to receive(:reussite?).and_return(true)
+        expect(restitution).to receive(:version?).and_return(false)
+        expect(restitution).to receive(:nombre_ouverture_contenant).and_return(300)
+        expect(
+          described_class.new(restitution).niveau
+        ).to eql(Competence::NIVEAU_1)
+      end
     end
   end
 
-  context 'en réussite et plus de 250 ouvertures de contenants' do
-    it 'a le niveau 2' do
-      expect(restitution).to receive(:reussite?).and_return(true)
-      expect(restitution).to receive(:nombre_ouverture_contenant).and_return(300)
-      expect(
-        described_class.new(restitution).niveau
-      ).to eql(Competence::NIVEAU_1)
+  describe "pour la version 2 de l'inventaire" do
+    before { expect(restitution).to receive(:reussite?).and_return(true) }
```

Comment:
ajouter un contexte "en cas de réussite" encapsulant les deux describe de version puis ajouter le before ci-dessus dans ce contexte

Reply:
C'est étrange de mettre un `expect` dans un before.

Cela veut dire qu'on est en train de tester quelque chose dans un before.

Il ne faudrait pas plutot faire un stub : 

```suggestion
    before { restitution.stub(:reussite?).and_return(true) }
```